how-to-guides/cloud-services/aws/connect-to-a-documentdb-cluster.md

```diff
@@ -0,0 +1,96 @@
+# Amazon DocumentDB
+Connecting to DocumentDB running in the private subnet of AWS VPC from DevBox.
+
+## Architecture Diagram:
+
+![image](../../../.gitbook/assets/documentdb-architecture.png)
+
+## Existing Database
+
+### Step 1: Accessing DocumentDB from a DevBox
+
+1. Follow the [Connecting to AWS](../../existing-network/connecting-to-aws.md) guide.
+2. Go to **Amazon DocumentDB > Clusters > Your Cluster.**
+3. Select **Connectivity & Security.** You will see the commands and DocumentDB hostname for accessing it.
+
+![image](../../../.gitbook/assets/documentDB-connectivity-and-security.png)
+
+4. Go to **DevBox**.
+5. Import the MongoDB public GPG key:
```

Comment:
Merge the two steps together:
In your DevBox, import the MongoDB public GPG key